buddyboss/buddyboss-platform
# Fixed misaligned "Select Folder/Album" button in Photo and document uploader popup

### All Submissions:


Fixes #2107 

### How to test the changes in this Pull Request:

1. Add FontAwesome
2. Check photo or document uploader in group or profile.

### Proof Screenshots or Video
https://prnt.sc/y5tyct
https://prnt.sc/y5toyh

### Other information:
